src/activities/fetch-review-content.ts

```diff
@@ -29,6 +29,18 @@ export const fetchEvaluationContent = async (evaluation: Evaluation): Promise<st
   throw Error('Could not resolve evaluation content URLs to content');
 };
 
+const mapEvaluation = async (evaluation: Evaluation): Promise<EPPEvaluation> => ({
+  date: evaluation.date,
+  doi: evaluation.doi,
+  participants: evaluation.participants.map<EPPParticipant>((participant): EPPParticipant => ({
+    name: participant.name,
+    role: participant.role,
+    institution: [participant.institution.name, participant.institution.location].filter((v) => v).join(', '),
```

Comment:
When we chatted earlier this approach was mentioned but ultimately discouraed because of the potential duplication of cities across instituation name and location. If we take this approach expect to see a lot of repeated city names in the instituation presentation. 

I did a quick regex on the docmap index and found 186 examples of this were found. I will drop in a few examples:

```
{"type":"organization","name":"University of California, Irvine","location":"Irvine, United States of America"}

{"type":"organization","name":"University of California, Davis","location":"Davis, United States of America"}

{"type":"organization","name":"University of California, Los Angeles","location":"Los Angeles, United States of America"}
```

Reply:
You're right, sorry. I'll update this to remove location.